README.md

```diff
@@ -14,7 +14,8 @@ The site was built for the CS98 course at Dartmouth College.
 
 ## Installation
 1. Clone repo to computer
-2. Install all requirements from requirements.txt
+2. Install postgresql (e.g. ```brew install postgresql```)
```

Comment:
Not yet but sourcemash doesn't run unless postgesql is installed for now. Heroku related?